diff --git a/lib/Planetscale/Postgres/PostgresDatabase.js b/lib/Planetscale/Postgres/PostgresDatabase.js
--- a/lib/Planetscale/Postgres/PostgresDatabase.js
+++ b/lib/Planetscale/Postgres/PostgresDatabase.js
@@ -23,7 +23,7 @@
     const getBr = yield* ops.getBranch;
     const createBr = yield* ops.createBranch;
     return {
-        stables: ["id", "organization", "kind", "region"],
+        stables: ["id", "organization", "region"],
         diff: Effect.fn(function* ({ news, olds, output }) {
             if (!isResolved(news))
                 return undefined;
